CHANGELOG.md

```diff
@@ -107,10 +107,26 @@ Adding a new version? You'll need three changes:
 
 ### Deprecated
 
-- Removed support for extensions/v1beta1 Ingress which was removed in kubernetes 1.22.
+- Removed support for `extensions/v1beta1` `Ingress` which was removed in kubernetes 1.22.
   At the same time deprecate `--enable-controller-ingress-extensionsv1beta1` CLI flag.
   [#3710](https://github.com/Kong/kubernetes-ingress-controller/pull/3710)
 
+- Removed support for `networking.k8s.io/v1beta1` `Ingress` which was removed in kubernetes 1.22.
+  At the same time deprecate `--enable-controller-ingress-networkingv1beta1` CLI flag.
+  [#3867](https://github.com/Kong/kubernetes-ingress-controller/pull/3867)
+
+  This changes the:
+  - Names of Kong services generated from Ingress v1.
+    From now on, they will contain the `pnum-` prefix in the part designating
+    the service port.
+    So, for example, instead of `default.foo-svc.80` we'll get
+    `default.foo-svc.pnum-80`.
```

Comment:
According to our agreements on naming pattern changes, such changes should be rolled out with the feature gate. Any specific reason to change this pattern for v1 when deprecating `v1beta1`? Maybe we could avoid this?

Reply:
Now that I think about this CHANGELOG entry it shouldn't really be stating this. This reflects the change that I needed to make in tests where I've changed netv1beta1 `Ingress`es into netv1 `Ingress`es.

The logic of [translating `Ingress` netv1 into kong configuration](https://github.com/Kong/kubernetes-ingress-controller/blob/4e3122af7e54028a82376614dd9ac79456d71a14/internal/dataplane/parser/translate_ingress.go#L33-L219) didn't change nor did the service backend port to string routine: https://github.com/Kong/kubernetes-ingress-controller/blob/4e3122af7e54028a82376614dd9ac79456d71a14/internal/dataplane/parser/translate_ingress.go#L21-L26

In such case, I'll remove this entry from changelog and I believe there's no need for feature gate then.

Reply:
That makes sense 👍

Reply:
Done